src/components/Executions/filters/useUserByFilterState.ts

```diff
@@ -0,0 +1,53 @@
+import { FilterOperationName } from 'models/AdminEntity/types';
+import { useEffect, useState, useContext } from 'react';
+import { FilterByUserState } from './types';
+import { useFilterButtonState } from './useFilterButtonState';
+import { APIContext } from 'components/data/apiContext';
+
+interface FilterByUserStateArgs {
+    queryStateKey: string;
```

Comment:
`queryStateKey` is used by filters to keep the value of the filter in sync with the URL in the address bar. You don't appear to be using this anywhere. So I would say either take it out, or implement the query state functionality.
[Here is how it is done in the SingleFilter](https://github.com/flyteorg/flyteconsole/blob/master/src/components/Executions/filters/useSingleFilterState.ts#L41)
I think you would want a query state key something like `currentUserOnly` and its value would be a `boolean`.